### PR TITLE
Non-connect scenarios

### PR DESCRIPTION
**Pull request by microbit-matt-hillsdon**
Tweak action enablement and placeholder screen to allow use of actions when the tool isn't connected.
This will help in import scenarios and allow some initial exploration without a micro:bit.

We still think there's a gap in basic introduction prior to the connect flow but will review separately.

### CODE REVIEW

File: src/components/DataSampleGridView.tsx [hunk 1]
```diff
@@ -56,37 +76,78 @@ const DataSamplesGridView = () => {
...
-      </Grid>
+      {gestures.length === 0 ? (
+        <VStack
+          gap={5}
+          flexGrow={1}
+          alignItems="center"
+          justifyContent="center"
+        >
+          <LoadProjectInput ref={loadProjectInputRef} accept=".json" />
+          <Text fontSize="lg">No data samples</Text>
```
Comment (microbit-matt-hillsdon) [inline src/components/DataSampleGridView.tsx:105]: This and the copy below needs to be extracted for translation

Comment (microbit-grace) [reply src/components/DataSampleGridView.tsx]: [d863bf2](https://github.com/microbit-foundation/ml-trainer/pull/331/commits/d863bf2e4d9beacfe922fcc205dd4ac177004988)

File: src/components/DataSampleGridView.tsx [hunk 2]
```diff
@@ -56,37 +77,80 @@ const DataSamplesGridView = () => {
...
+          </Text>
+          {!isConnected && (
+            <Text fontSize="lg">
+              <Button
+                fontSize="lg"
+                color="brand.600"
+                variant="link"
+                onClick={actions.startConnect}
+              >
+                Connect a micro:bit
```
Comment (microbit-matt-hillsdon) [inline src/components/DataSampleGridView.tsx:117]: We need to extract this whole thing with link1/link2 tags.

Comment (microbit-grace) [reply src/components/DataSampleGridView.tsx]: [320b30a](https://github.com/microbit-foundation/ml-trainer/pull/331/commits/320b30abde8c942526e796505d5f3a091411425d)

File: src/components/DataSampleGridView.tsx (hunk 1 above)
Comment (microbit-grace) [reply src/components/DataSampleGridView.tsx]: Oops, I didn't read the message properly and missed "the copy below needs to be extracted for translation"

File: src/components/DataSampleGridView.tsx [hunk 3]
```diff
@@ -1,44 +1,65 @@
...
 ];
 
 const DataSamplesGridView = () => {
   const gestures = useStore((s) => s.gestures);
   const [selectedGestureIdx, setSelectedGestureIdx] = useState<number>(0);
-  const selectedGesture = gestures[selectedGestureIdx] ?? gestures[0];
+  const selectedGesture: GestureData | undefined =
+    gestures[selectedGestureIdx] ?? gestures[0];
+
+  // TODO: check
```
Comment (microbit-grace) [inline src/components/DataSampleGridView.tsx:49]: Is there anything we need to check?
```suggestion
```

Comment (microbit-matt-hillsdon) [reply src/components/DataSampleGridView.tsx:49]: I guess the gestures.length === 0 part isn't really right any more but not consequential.

**Review by microbit-grace — APPROVED** [review]

**Pull request merged**